ssdaq/core/ss_data_classes.py

```diff
@@ -8,6 +8,19 @@
 N_BYTES_NUM = 8 #Number of bytes to encode numbers (uint and float) in the SSReadout
 N_CAM_PIX = N_TM*N_TM_PIX #Number of pixels in the camera
 
+from collections import namedtuple as _nt
+ 
+_SSMappings = _nt('SSMappings','ssl2colrow ssl2asic_ch')
```

Comment:
Is this the best place to store the ssl2asic_ch conversion array (if we are to apply the conversion in software instead of firmware)? An alternative is storing it in TargetCalib as its own file alongside the pixel mapping. A version can then be attached to the conversion array, making it robust to changes in the ordering in future camera versions.

Does this package already have dependencies on the TARGET Software, or not?

Reply:
No currently it does not have any dependencies on the target software I was thefore reluctant to put it there. In the longer run I would like to have a dataclass/mapping project that would provide the common data structures and mappings used in CHEC

Reply:
Agreed - TargetCalib is not the correct long-term location for it